locale/en/knowledge/advanced/buffers/how-to-use-buffers.md

```diff
@@ -61,19 +61,25 @@ This initializes the buffer to a binary encoding of the first string as specifie
 
 Given that there is already a buffer created:
 
-    > var buffer = new Buffer(16);
+```
+> var buffer = new Buffer(16);
```

Comment:
Can you try with `bash` in such cases? Is it highlighted properly?

Reply:
Actually scratch that, this should happen in another PR. Just make sure any languages you specify exist in prism.

Reply:
No, bash wouldn't work since this is the Node REPL